Jenkinsfile

```diff
@@ -66,24 +74,6 @@ pipeline {
                 '''
             }
         }
-
-        stage('Lint Integration Tests') {
-            steps {
-                sh './bin/lint_tests'
```

Comment:
Runs in Runs in https://github.com/cyberark/conjur-openapi-spec/blob/main/.github/workflows/linting.yml

@telday can we delete the `bin/lint_tests` script, or do we want to leave it for convenience?

Maybe the action should run it exactly, so that the pipeline does the same thing as the manual version?

Reply:
I would prefer to leave it for convenience, I regularly run it locally when I make changes to the tests. I updated the action to just run the test, don't know why it was set up like it was before